parts/kubernetesmasteraddons-managed-azure-storage-classes.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: default
+  annotations:
+    storageclass.beta.kubernetes.io/is-default-class: "true"
+  labels:
+    kubernetes.io/cluster-service: "true"
+provisioner: kubernetes.io/azure-disk
+parameters:
+  kind: Managed
+  storageaccounttype: Standard_LRS
+---
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: managed-premium
+  annotations:
+  labels:
+    kubernetes.io/cluster-service: "true"
+provisioner: kubernetes.io/azure-disk
+parameters:
+  kind: Managed
+  storageaccounttype: Premium_LRS
+---
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: unmanaged-standard
```

Comment:
@andyzhangx I notice that the unmanaged yaml file sets this value to `managed-standard`. Is it intentional that the two `...-azure-storage-classes.yaml` manifests have these somewhat non-intuitive distinctions?

Reply:
@jackfrancis good catch, because the default storage class of `kubernetesmasteraddons-managed-azure-storage-classes.yaml` is already `managed-standard`. Anyway, the `unmanaged-standard` storage class is useless in managed cluster and `managed-standard` storage class is useless in unmanaged cluster.  I will change this storage class to `managed-standard`